src/2048-expert/index.html

```diff
@@ -13,6 +13,22 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+        <ul>
+            <li>Take actions with arrow keys</li>
```

Comment:
Maybe consider using line breaks instead of list bullet points to make the left margin align with the rest of the text. 

![image](https://user-images.githubusercontent.com/78716153/185070929-a9a67600-33c2-41e7-87bb-691db15a763b.png)